project/Publishing.scala

```diff
@@ -0,0 +1,35 @@
+import sbt.Keys._
+import sbt._
+
+/**
+  * NOTE: This was lifted wholesale from Cromwell.
+  */
+
+object Publishing {
+  private val buildTimestamp = System.currentTimeMillis() / 1000
+
+  private def artifactoryResolver(isSnapshot: Boolean): Resolver = {
+    val repoType = if (isSnapshot) "snapshot" else "release"
+    val repoUrl =
+      s"https://artifactory.broadinstitute.org/artifactory/libs-$repoType-local;build.timestamp=$buildTimestamp"
+    val repoName = "artifactory-publish"
+    repoName at repoUrl
+  }
+
+  private val artifactoryCredentials: Credentials = {
+    val username = sys.env.getOrElse("ARTIFACTORY_USERNAME", "")
+    val password = sys.env.getOrElse("ARTIFACTORY_PASSWORD", "")
+    Credentials("Artifactory Realm", "artifactory.broadinstitute.org", username, password)
+  }
+
+  val modelPublishSettings: Seq[Setting[_]] =
+    //we only publish to libs-release-local because of a bug in sbt that makes snapshots take
+    //priority over the local package cache. see here: https://github.com/sbt/sbt/issues/2687#issuecomment-236586241
+    Seq(publishTo := Option(artifactoryResolver(false)), credentials += artifactoryCredentials)
```

Comment:
slight difference to cromwell here in passing `false` rather than the sbt `isSnapshot` key, which doesn't work right